parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1544,7 +1516,7 @@ loop = do
         names <- makePrintNamesFromLabeled' $
           LD.referents testTerms <>
           LD.referents [ DD.okConstructorReferent, DD.failConstructorReferent ]
-        ppe <- prettyPrintEnv names
+        ppe <- fqnPPE names
```

Comment:
This is a spot where we want to see fully qualified names - the test result report, we want the test names to be their full path.